lib/mongoid/criteria/queryable/pipeline.rb

```diff
@@ -72,16 +72,24 @@ def project(entry)
         #
         # @example Add the unwind.
         #   pipeline.unwind(:field)
+        #   pipeline.unwind(document)
+        #
+        # @see https://docs.mongodb.com/manual/reference/operator/aggregation/unwind/
         #
         # @param [ String, Symbol ] field The name of the field.
```

Comment:
Hi @chenzww 
Would you mind updating the documentation too so we can merge this? Thanks!